docs/metrics/accelerate.md

```diff
@@ -9,80 +9,86 @@ Pulse calculates the four key Accelerate metrics using strictly **changes**, **d
 
-| Deployment frequency | Value                                                                                                                                  |
-| -------------------- | -------------------------------------------------------------------------------------------------------------------------------------- |
-| On-demand            | Median of more than one `deployments` per day (i.e. more than 50% of the days have more than one deployment.)                          |
-| Daily                | Median of three or more days with `deployments` in a calendar week (i.e. more than 50% of weeks have 3 or more days with deployments.) |
-| Weekly               | Median of one or more `deployments` in a calendar week (i.e. more than 50% of weeks have at least one deployment.)                     |
-| Monthly              | Median of one or more `deployments` in a calendar month (i.e. more than 50% of months have at least one deployment.)                   |
-| Yearly               | None of the above                                                                                                                      |
+Pulse uses the average value over a period of time to display aggregate deployment frequencies.
 
-!!! note
-    The value above is used to determine your Performance level. The numeric value displayed as Deployment Frequency is the number of `deployments` per day.
-
-When we aggregate Deployment Frequency we use the average of a period of time.
+Pulse determines your performance level for this metric as follows:
 
-| Performance level[^1] | Value                                                              |
-| --------------------- | ------------------------------------------------------------------ |
-| Low                   | Monthly or Yearly (Between once per month and once every 6 months) |
-| Medium                | Weekly (Between once per week and once per month)                  |
-| High                  | Daily (Between once per day and once per week)                     |
-| Elite                 | On-demand (Multiple deployments per day)                           |
+| Performance level[^1] | Deployment frequency                                                                                              |
+| --------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| Elite                 | On-demand (multiple deploys per day)<br/>(median of more than one **deployment** per day)                         |
+| High                  | Between once per day and once per week<br/>(median of three or more days with **deployments** in a calendar week) |
+| Medium                | Between once per week and once per month<br/>(median of one or more **deployments** in a calendar week)           |
+| Low                   | Between once per month and once every six months<br/>(median of one or more **deployments** in a calendar month)  |
```

Comment:
I propose that we simplify the two previous tables into a single one since the "qualitative deployment frequency" is not really shown on the Pulse UI, only the corresponding performance level.

I'm also dropping the explanation of what each median means (such as "more than 50% of the days have more than one deployment") because I felt that it added a lot of noise but we can add it back if you feel it's helpful.

Reply:
Yes, that will make it even more straightforward. :ok_hand: